### DISCUSSION TITLE
[Feat]: Prevent Overdrag/scroll

### DISCUSSION THREAD

**Discussion by tayteboss** [davidjerleke/embla-carousel#840] [Ideas]
### Which variants of Embla Carousel are relevant to this feature request?

- [X] embla-carousel **(Core)**
- [ ] embla-carousel-react
- [ ] embla-carousel-vue
- [ ] embla-carousel-svelte
- [ ] embla-carousel-autoplay
- [ ] embla-carousel-auto-scroll
- [ ] embla-carousel-solid
- [ ] embla-carousel-auto-height
- [ ] embla-carousel-class-names
- [ ] embla-carousel-docs **(Documentation)**
- [ ] embla-carousel-docs **(Generator)**

### Feature description

Is there a way to stop users from being able to overscroll/drag entirely? So if you hit the start/end of the carousel it's a hard wall and that slide becomes stuck? I appreciate the library is physics-based and I'm assuming it's the intention of the carousel however thought I would ask if it is possible.

Thanks

### Additional Context

Additional details here...

### Before submitting

- [X] I've made research efforts and searched the documentation
- [X] I've searched for existing issues
- [X] I agree to follow this project's Contributing Guidelines for feature requests

**Comment by davidjerleke** [comment]
@tayteboss there's no built-in way to solve this. Here's a hack that might solve this but I haven't had the time to check if it's up to date and works with the current version of Embla:

- https://github.com/davidjerleke/embla-carousel/issues/42#issuecomment-2018273079

This is not prioritised so I probably won't have time to update it if it needs an update.

Best,
David
